app/pages/BeelieversAuction/nft.tsx

```diff
@@ -92,75 +93,77 @@ export function NftDisplay({ nftId }: NftDisplayProps) {
-							🎉 {metadata.name || "Beeliever NFT"} Minted!
-						</h4>
-
-						<div className="space-y-2">
-							{nftType && (
-								<div className="flex items-center gap-2">
-									<span
-										className={`inline-flex items-center gap-1 px-2 py-1 text-xs rounded-full ${
-											nftType === "Mythic"
-												? "bg-gradient-to-r from-yellow-400/20 to-orange-400/20 text-yellow-400 border border-yellow-400/30"
-												: "bg-primary/20 text-primary border border-primary/30"
-										}`}
-									>
-										{nftType === "Mythic" ? "✨" : "🐝"} {nftType}
-									</span>
-									{mythicName && (
-										<span className="text-xs text-muted-foreground">{mythicName}</span>
-									)}
-								</div>
-							)}
+				<div className="w-full space-y-5 text-center">
+					<div className="space-y-3">
+						<div className="text-xl flex-col font-bold bg-gradient-to-r from-primary to-orange-400 bg-clip-text flex items-center justify-center gap-3">
+							<span>🎉 {metadata.name || "Beeliever NFT"}</span>
+							<span className="text-primary">Your NFT!</span>
+						</div>
 
-							{background && (
-								<div className="text-xs text-muted-foreground">Background: {background}</div>
-							)}
+						<div className="flex items-center justify-center gap-2 text-sm text-muted-foreground">
+							<span className="font-medium">Beeliever #{metadata.token_id}</span>
+						</div>
+						{nftType && (
+							<div className="flex flex-wrap items-center justify-center gap-2">
+								<span
+									className={`inline-flex items-center gap-2 px-2 py-1 text-sm font-medium rounded-full transition-all duration-300 hover:scale-105 ${
+										nftType === "Mythic"
+											? "bg-gradient-to-r from-yellow-400/20 to-orange-400/20 text-yellow-400 border border-yellow-400/30 shadow-lg"
+											: "bg-gradient-to-r from-primary/20 to-blue-400/20 text-primary border border-primary/30 shadow-md"
+									}`}
+								>
+									{nftType === "Mythic" ? "✨ " : "🐝 "}
+									{nftType}
+									{mythicName ? ": " + mythicName : ""}
+								</span>
+							</div>
+						)}
+						<div className="flex items-center justify-center gap-2 text-sm text-muted-foreground">
+							<span className="font-medium">Object ID: {trimAddress(nftId)}</span>
+						</div>
+					</div>
+					<div className="space-y-3">
+						{background && (
+							<div className="flex items-center justify-center gap-2 text-sm">
+								<span className="text-muted-foreground">Background:</span>
+								<span className="font-medium">{background}</span>
+							</div>
+						)}
 
-							{metadata.badges && metadata.badges.length > 0 && (
-								<div className="flex flex-wrap gap-1">
+						{metadata.badges && metadata.badges.length > 0 && (
+							<div className="space-y-2">
+								<span className="text-sm font-medium text-muted-foreground">Badges:</span>
+								<div className="flex flex-wrap justify-center gap-2 mt-2">
```

Comment:
I think you don't need so many wrapping! Span are inline, no need to make flex here.